Source/GoBot/ViewDatabase.swift

```diff
@@ -187,7 +187,7 @@ class ViewDatabase {
         try db.execute("PRAGMA journal_mode = WAL;")
         
         
-//        db.trace { print("\tSQL: \($0)") } // print all the statements
+        db.trace { print("\tSQL: \($0)") } // print all the statements
```

Comment:
Please, comment it again as checking the unit tests results are painful with all those logs

Reply:
Fixed